webpack.prod.js

```diff
@@ -7,7 +7,7 @@ module.exports = merge(common, {
     externals: {
         config: JSON.stringify({
             networkId: 4,
-            marketAddress: "0x62de2c2707519b32b8fa1a7a554ab7863130a056",
+            marketAddress: "0xcb3eaafa1a9335d2993c3021b8e089150a0233fd",
```

Comment:
Is this another deployment you've made??

Reply:
It looks like the older address to me, if so please change it back to the newer one